vip-helpers/vip-utils.php

```diff
@@ -1058,8 +1058,16 @@ function wpcom_vip_load_plugin( $plugin = false, $folder = false, $load_release_
 
 		return _wpcom_vip_include_plugin( $includepath );
 	} else {
+		$error_msg = "Unable to load plugin '$plugin' using wpcom_vip_load_plugin()!";
```

Comment:
It might be good to fix up the error a bit:

```
$error_msg = sprint( 'wpcom_vip_load_plugin: Unable to load plugin `%s`; the path `%s` does not exist.', $plugin, $includepath );
```

Reply:
Good point. Done.